lib/l10n/app_fr.arb

```diff
@@ -53,33 +53,33 @@
   },
   "noServersAvailable": "Aucun serveur disponible",
   "error": "Erreur",
-  "videoError": "An error happened while trying to play the video.",
+  "videoError": "Une erreur est survenue lors de la lecture de la vidé.",
   "ok": "OK",
-  "retry": "Retry",
+  "retry": "Réessayer",
   "removeCamera": "Enlever caméra",
   "replaceCamera": "Remplacer caméra",
   "reloadCamera": "Recharger caméra",
   "selectACamera": "Sélectionner une caméra",
   "switchCamera": "Switch camera",
   "online": "En ligne",
   "offline": "Hors ligne",
-  "live": "LIVE",
-  "timedOut": "TIMED OUT",
-  "loading": "LOADING",
-  "recorded": "RECORDED",
+  "live": "EN DIRECT",
+  "timedOut": "EXPIRÉ",
+  "loading": "CHARGEMENT",
+  "recorded": "ENREGISTREMENT",
   "removeFromView": "Retirer de la vue",
   "addToView": "Ajouter à la vue",
-  "addAllToView": "Add all to view",
-  "removeAllFromView": "Remove all from view",
-  "eventBrowser": "Historique d'événements",
-  "eventsTimeline": "Timeline of Events",
+  "addAllToView": "Tout ajouter à la vue",
+  "removeAllFromView": "Tout retirer de la vue",
+  "eventBrowser": "Navigateur d'événements",
+  "eventsTimeline": "Ligne du temps",
   "server": "Serveur",
   "device": "Appareil",
   "event": "Évènement",
   "duration": "Durée",
   "priority": "Priorité",
-  "next": "Next",
-  "previous": "Previous",
+  "next": "Suivant",
+  "previous": "Précédent",
   "lastImageUpdate": "Last Image Update",
```

Comment:
Forgot this one?

Reply:
Yup, it would be "Dernier rafraîchissement"

Reply:
"lastImageUpdate": "Last Image Update",